### ISSUE TITLE
Check for Smallpeice RUB in Newbury

### ISSUE THREAD

**Issue by trickeydan** [srobo/tasks#288]
It's possible that a RUB of Smallpeice equipment may be in the container.

Next time someone visits the container, it would be useful if this was checked and confirmed.

**Comment by PeterJCLaw** [comment]
How would one identify this?

**Comment by trickeydan** [comment]
@PeterJCLaw

An unlabeled RUB, possibly without an asset code, full of packets addressed to ECS. I believe it was taken by mistake last time there was a kit sorting event at Soton.

**Comment by RealOrangeOne** [comment]
For clarification, is this likely to have been one of the transparent RUBs SRO has?

I am 100% sure there were no transparent RUBs taken to the container.

**Comment by trickeydan** [comment]
@RealOrangeOne It was a white one. SRO have both types.

**Comment by PeterJCLaw** [comment]
@andybusse and I were at the container today. We looked for this RUB and couldn't find it. All the RUBs which are container definitely have SR part codes at this point, which seems to rule out the SRO RUB being there.

I also looked in one of our RUBs which I thought might have been a candidate for having SRO stuff in it (the RUB from Munich), but it didn't seem to have anything other than what I'd expect to be in it.
